front/src/applications/stdcmV2/components/StdcmDestination.tsx

```diff
@@ -24,6 +24,13 @@ const StdcmDestination = ({
 }) => {
   const { t } = useTranslation('stdcm');
   const dispatch = useAppDispatch();
+
+  const [arrivalDateInput, setArrivalDateInput] = useState<Date | undefined>(undefined);
+  const [arrivalTimeInput, setArrivalTimeInput] = useState<{ hours: number; minutes: number }>({
+    hours: 0,
+    minutes: 0,
+  });
+
```

Comment:
I noticed that these states are duplicated in `StdcmDestination` and `StdcmOrigin`, so we might want to move them up to the parent `StdcmConfig`. Then, we can pass the necessary data to `StdcmDestination` and `StdcmOrigin`. 

To simplify things, I suggest unifying the variables. Instead of having `arrivalDateInput` and `arrivalTimeInput`, we could use a single variable called `scheduleConstraint` (you can suggest another name). This could have the following type:

```ts
type ScheduleConstraint = {
    date: Date;
    hour: number;
    minutes: number;
}
```

Then, `StdcmConfig` can manage this new state variable (`scheduleConstraint`) and pass it to `StdcmDestination` and `StdcmOrigin`.